service/src/main/java/io/camunda/service/search/filter/UserTaskFilter.java

```diff
@@ -35,9 +35,9 @@ public final record UserTaskFilter(
     implements FilterBase {
 
   public static final class Builder implements ObjectBuilder<UserTaskFilter> {
-    private List<Long> userTaskKeys;
-    private List<String> userTaskDefinitionIds;
-    private List<String> processNames;
+    private List<Long> keys;
+    private List<String> elementIds;
+    private List<String> bpmProcessDefinitionIds;
```

Comment:
❓ Where is this name coming from, why are we changing to this name?

Reply:
My idea over change this name is have consistency over the other names as well. 
Over the service layer we call userService(), and we know the key is related to the user task

related to the elementsIds and bpmProcessDefinitionIds, the goal is have the same names as the responses. 

Do you have a different aspect over here?

Reply:
OK, I see, I was mostly wondering about the `bpmProcessDefinitionIds`, sorry for not being clear enough (I only marked that attribute but GitHub displays the lines before that as well). I tried to find the term `bpmProcessDefinitionId` somewhere else in the existing codebase but couldn't find anything. So far, I can only find `bpmnProcessId` but no `bpmProcessDefinitionId`. Can you tell me why you chose `bpmProcessDefinitionId` explicitly? Do we use this somewhere else as well?

Reply:
You have reason over here
This should be `bpmnProcessId`, Just recheck in the document. 
I had a mistaken with the V1 Tasklist API that we were using <attribute>DefinitionId

Reply:
OK, no worries, not a big deal and change, I just wanted to make sure I didn't miss anything 😄